client/app/bundles/course/admin/pages/RagWiseSettings/components/ForumList.tsx

```diff
@@ -0,0 +1,67 @@
+import { FC, memo } from 'react';
+import { List, Typography } from '@mui/material';
+import equal from 'fast-deep-equal';
+
+import Section from 'lib/components/core/layouts/Section';
+import { useAppSelector } from 'lib/hooks/store';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import { EXPAND_SWITCH_TYPE } from '../constants';
+import { getAllCourses } from '../selectors';
+import translations from '../translations';
+
+import ExpandAllSwitch from './buttons/ExpandAllSwitch';
+import CourseTab from './CourseTab';
+
+const ForumList: FC = () => {
+  const { t } = useTranslation();
+  const courses = useAppSelector((state) => getAllCourses(state));
+
+  return (
+    <Section
+      contentClassName="flex flex-col space-y-3"
+      sticksToNavbar
+      subtitle={t(translations.forumSectionSubtitle)}
+      title={t(translations.forumSectionTitle)}
+    >
+      {courses.length === 0 ? (
+        <div className="flex justify-center items-center h-full">
+          <Typography align="center" variant="body1">
+            {t(translations.noRelatedCourses)}
+          </Typography>
+        </div>
+      ) : (
+        <section>
+          <div className="flex justify-between items-center mb-4">
+            <div>
+              <ExpandAllSwitch type={EXPAND_SWITCH_TYPE.courses} />
+            </div>
+            <div className="pr-5 space-x-48 flex justify-end">
+              <div className="text-center">
```

Comment:
can't you apply text-center on the parent or the `Typography` instead?

Reduce nesting of elements.

Reply:
followed convention in `client/app/bundles/course/admin/pages/CodaveriSettings/components/AssessmentList.tsx:70`. But I can remove it.